error.go

```diff
@@ -0,0 +1,12 @@
+package currency
+
+import "errors"
+
+// errorInvalidISO : returns an error for an invalid ISO code
+var errorInvalidISO = errors.New("Error: Invalid ISO Code")
```

Comment:
No need to privatize these errors. If the end user wants to do error checks like if err == errInvalidISO they would be able to do so and would have be forced to write out the full string. Also remove Error: